docs/configuration.md

```diff
@@ -10,10 +10,10 @@ Honeycomb uses environment variables during build and run time to control the di
 * `ELECTRON_START_URL` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow): URL (e.g. `http://localhost:3000`) where the front end of the app is being hosted - also used in `electron.js` to indicate the app is running in dev mode
 * `EVENT_MARKER_PRODUCT_ID`![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The product ID of the event marker (e.g. `0487`).  If not set, it will use the `productID` set in `public/config/trigger.js` if available, or attempt to connect using the com name.
 * `EVENT_MARKER_COM_NAME` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The com name of the event marker (e.g. `COM3`). If not set, it will use the `comName` set in `public/config/trigger.js`.  If the productID is set (not an empty string), this field will be ignored.
-* `REACT_APP_VOLUME`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being asked to adjust volume. This can be used when the task is running as a desktop app (electron). Not defining this variable is equivalent to setting it to `false`. Build-time and run-time values need to be the same.
-* `REACT_APP_VIDEO`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-runtime-purple)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being video recorded. This can be used on both the desktop (electron) and online settings. Not defining this variable is equivalent to setting it to `false`.
+* `REACT_APP_VOLUME`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being asked to adjust volume. This can be used on both the desktop (electron) and online settings. Not defining this variable is equivalent to setting it to `false`. 
```

Comment:
should all of these env vars have the optional badge (is there a non-optional env var?)

Reply:
How about changing the optional badge be changed to default value?

Reply:
How would a default badge work?

Reply:
You write a link with the made up URL e.g https://img.shields.io/badge/default-false-yellow ![](https://img.shields.io/badge/default-false-yellow)

Reply:
Do I still have to write the "Not defining this variable is equivalent to setting it to `false`. " in this case?

Reply:
Not necessary then

Reply:
Agreed, but I'd add the alt test for all of these badges so it's more accessible (e.g. `![default false](https://img.shields.io/badge/default-false-yellow)`)

Reply:
same for the runtime/buildtime and any other badges if we aren't reiterating that info in the description